berkeley-mobile/Map/MapMarkerDetailView.swift

```diff
@@ -120,6 +119,12 @@ class MapMarkerDetailView: UIView {
         detailStack.alignment = .center
     
         verticalStack.bottomAnchor.constraint(equalTo: contentView.layoutMarginsGuide.bottomAnchor).isActive = true
+        
+        //swipe down gesutre
```

Comment:
typo, also you don't really need to comment this because the code is pretty self-explanatory here